File: week15/p_62048.py
def gcb(w,h):
    a,b = max(w,h), min(w,h)
    while True:
        r = a % b
        if r == 0:
            return b
        else:
            a = b
            b = r
    
def solution(w,h):
    answer = w*h -(w + h - gcb(w,h))
    return answer

File: week19/p_62048.js

function solution(w, h) {
    
    function gcd(w,h){
        a= Math.max(w,h)
        b = Math.min(w,h)
        while(a){
            r= a% b
            if (r ===0){
                return b
            }else {
                a = b
                b = r
            }
        }
        
    }
    let answer = (w*h) -(w+h-gcd(w,h) )
    return answer;
}